Lib/operator.py

```diff
@@ -239,7 +239,7 @@ class attrgetter:
     """
     __slots__ = ('_attrs', '_call')
 
-    def __init__(self, attr, *attrs):
+    def __init__(self, attr, /, *attrs):
```

Comment:
Technically, a signature change can affect *some* people, but it should be fine, because:
1. most people use C version by default
2. docs never promised that

Now C and Python signatures are the same.